test/SlowTests/Issues/RavenDB-5241.cs

```diff
@@ -1,108 +0,0 @@
-// -----------------------------------------------------------------------
-//  <copyright file="RavenDB-5241.cs" company="Hibernating Rhinos LTD">
-//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
-//  </copyright>
-// -----------------------------------------------------------------------
-
-using System.Linq;
-using FastTests;
-using Raven.Client.Documents.Indexes;
-using Raven.Client.Documents.Transformers;
-using Xunit;
-
-namespace SlowTests.Issues
-{
-    public class RavenDB_5241 : RavenTestBase
-    {
-        [Fact]
-        public void loading_documents_with_transformer_duplicate_ids()
```

Comment:
Restore candidate: duplicated ids send to the server, results don't contain duplicates